modules/publication/php/publication.class.inc

```diff
@@ -0,0 +1,251 @@
+<?php
+/**
+ * Publications module class
+ *
+ * PHP version 7
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   David Blader  <dblader.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/mohadesz/Loris-Trunk
+ */
+namespace LORIS\publication;
+
+/**
+ * Module for viewing research project proposals and their respective publications
+ *
+ * @category Behavioural
+ * @package  Main
+ * @author   David Blader  <dblader.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class Publication extends \NDB_Menu_Filter_Form
+{
+    public $AjaxModule   = true;
```

Comment:
Can you add comments explaining why these need to stay public. I feel like it would be easy for a future dev to say "hey thats not safe" and then switch it

Reply:
i do not have a particularly good reason. in parent class `NDB_Menu_Filter` it's declared with `var` which is equivalent to `public` and i vaguely remember learning about some OOP principle about how child classes should not change the visibility of parent class attributes

Reply:
I guess just a comment with that is fine then. Unless someone else can explain why?